doc/source/development/contributing.rst

```diff
@@ -295,7 +295,7 @@ Below is a brief overview on how to set-up a virtual environment with Powershell
 under Windows. For details please refer to the
 `official virtualenv user guide <https://virtualenv.pypa.io/en/stable/userguide/#activate-script>`__
 
-Use an ENV_DIR of your choice. We'll use ~\virtualenvs\pandas-dev where
+Use an ENV_DIR of your choice. We'll use ~\<span class="x x-first x-last">\</span>virtualenvs\\pandas-dev where
```

Comment:
@WillAyd, the putting span tag was not my own action. I simply accepted — I think your — recommendation by clicking a button.

In my [cloned Git repository](https://github.com/MarianD/pandas/commits/doc-dev-contrib) there is a note in this commit:

> Co-Authored-By: William Ayd <william.ayd@icloud.com>

Reply:
```suggestion
Use an ENV_DIR of your choice. We'll use ~\\virtualenvs\\pandas-dev where
```

Hmm that's interesting. Maybe GH is just escaping it on its own. 

Can try again, but otherwise might want to fix locally and repush. Thanks!

Reply:
@WillAyd, your explanation is probably right, thanks. I did as you suggested — fixed it locally and repushed.